code/src/index.css

```diff
@@ -11,3 +11,96 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.date {
+  position: absolute;
+  right: 18%;
+  font-style: italic;
+  
+}
+
+.main-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.happy-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 10px;
+  margin: 10px;
+  width: 80%;
+  border: 2px solid black;
+  box-shadow: 6px 6px;
+  padding: 10px;
+  margin: 10px;
+  background: lightgray;
+
+}
+
+.happy-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 80%;
+  height:auto;
+  border: 2px solid black;
+  box-shadow: 6px 6px;
+  padding: 10px;
+  margin: 10px;
+  word-wrap: break-word;
+}
+
+.submit-button {
+  background: salmon;
+  border-radius: 15px;
+  padding: 10px;
+  margin: 10px;
+  display: flex;
+  flex-wrap: nowrap;
+
+}
+
+.message-field {
+  margin: 15px;
+  padding: 10px;
+  height: 50px;
+  width: 70%;
+}
+
+.heart-button {
+  
+  border: none;
+  outline:none;
+  background: white;
+}
+
+.heart-button-active {
+  
+  border-radius: 100%;
+  background: rgb(248, 143, 143);
+  padding: 12px;
+}
+
+.heart-button-not-active {
+  
+  border-radius: 100%;
+  padding: 12px;
+  background: lightgray;
+}
+
+.button-wrapper {
+  display: flex;
+  flex-direction: row;
+  padding: 10px;
+  margin: 10px;
+}
+
+.heart-count {
+  margin: 10px;
+  padding: 10px;  
+}
+
```

Comment:
You really know how to name the classes! Your CSS-file is easy to understand when your classnames are descriptive.